docs/src/app/components/pages/components/SVGIcon/README.md

```diff
@@ -0,0 +1,15 @@
+## SVG Icons
+Using mui.SvgIcon, we can create a custom [svg component](https://www.google.com/design/spec/style/icons.html#). Here we are creating the ActionHome SvgIcon for this docs site, and using it in some separate component.
+Custom SvgIcon components can be included as children for other Material UI components that use icons such as [IconButtons](http://www.material-ui.com/#/components/icon-buttons).
+
+For all of the Google's Material Design Icons prebuilt SvgIcon components exist. They can be used this way:
```

Comment:
I think that we can remove this example from the readme and add it to :
docs/src/app/components/pages/components/SVGIcon/ExampleSimple.jsx